tribe-ext-eventbrite-addl-opts/index.php

```diff
@@ -188,10 +174,17 @@ public function after_iframe() {
 		return $text;
 	}
 
-	// originally from https://theeventscalendar.com/knowledgebase/changing-the-height-of-the-eventbrite-tickets-iframe/
+	/**
+	 * Replace the iframe's inline styling height within the ticket area HTML
+	 *
+	 * @param $html
+	 * @param $event_id
+	 * @param $post_id
+	 *
+	 * @return mixed|string
```

Comment:
For each of your public functions that are attach to a filter, maybe you could replace the `@param` and `@return` with `@see`? Generally speaking blank doc blocks are quite frowned on. With extensions I'm sure it can slide a little more often, but, at the very least when documenting a filter it makes sense to `@see` instead of a blank docblock.